webapi/tct-backgrounds-css3-tests/tests.full.xml

```diff
@@ -6551,7 +6551,7 @@
       </testcase>
       <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Backgrounds and Borders Module Level 3" execution_type="manual" id="background-origin-002" priority="P2" purpose="Check if background-origin with initial value display correctly in visual" status="approved" type="compliance">
         <description>
-          <test_script_entry timeout="90">/opt/tct-backgrounds-css3-tests/backgrounds/csswg/background-origin-002.html</test_script_entry>
+          <test_script_entry timeout="90">/opt/tct-backgrounds-css3-tests/backgrounds/csswg/background-origin-002-manual.html</test_script_entry>
```

Comment:
Run  ref  tests automatically need webdriver , if no webdriver, we must run these manually. Same as bdd tests.